doctrine/doctrine-bundle/1.6/config/packages/doctrine.yaml

```diff
@@ -7,14 +7,6 @@ parameters:
 
 doctrine:
     dbal:
-        # configure these for your database server
-        driver: 'pdo_mysql'
-        server_version: '5.7'
```

Comment:
`serverVersion` should rather be specified in DATABASE_URL as the platform may be different in different envs (different developers, servers etc.) - having it in VCS introduces a environment lock-in smell.

Reply:
None of the cloud hostings for Postgresql that I know of are providing it in their credentials URL when using heroku addons (I haven't checked their automatic integration with other PaaS).

And having different server versions on different environments can cause issues in the project anyway (the schema queries generated for doctrine/migrations might be different for different versions of the platform, as new versions might use new features)

Reply:
> None of the cloud hostings for Postgresql that I know of are providing it in their credentials URL

But this bundle is configuration for DBAL and DBAL does support it. Also cloud deployments is just a portion of usages.

> And having different server versions on different environments can cause issues in the project anyway

It can, it may not. I've personally haven't seen any dealbreaker since 9.4.

> the schema queries generated for doctrine/migrations might be different

True. But this was also true for 9.4.7->9.4.8 few years ago. Also this is only applicable to generating Migrations and has no impact on runtime.

> Upgrading Postgresql to a new major version may require project changes anyway.

We successfully migrated multiple 9.6 instances to 10 without changing a single line of code, the only needed _change_ was bumping DBAL to 2.7. :)